contrib/export/types/types.go

```diff
@@ -0,0 +1,13 @@
+package types
```

Comment:
Seems redundant for just one file right? Why not keep this under `contrib/export/types.go`

Reply:
I was expecting this to grow a bit more